src/components/Wallet/Wallet.tsx

```diff
@@ -0,0 +1,103 @@
+import {useEffect, useState} from "react";
+ import {
+   ETHTokenType,
+   ImmutableXClient,
+   Link
+ } from "@imtbl/imx-sdk";
+
+ enum WalletConsts {
+   WALLET_ADDRESS = "WALLET_ADDRESS",
+   STARK_PUBLIC_KEY = "STARK_PUBLIC_KEY",
+   LINK_URL = "https://link.ropsten.x.immutable.com",
+   API_URL = "https://api.ropsten.x.immutable.com/v1",
+   ETH_NETWORK = "ETH_NETWORK",
+ }
+
+ const Wallet = () => {
+   const link = new Link(WalletConsts.LINK_URL);
```

Comment:
Consider making all the states and state management reused here into a custom hook
https://reactjs.org/docs/hooks-custom.html